meinberlin/apps/plans/assets/ListMapBox.jsx

```diff
@@ -81,11 +81,9 @@ class ListMapBox extends React.Component {
   }
 
   isInTitle (title) {
-    let titleLower = title.toLowerCase()
-    if (titleLower.indexOf(this.state.titleSearch.toLowerCase()) === -1) {
-      return false
-    }
-    return true
+    let titleLower = title.toLowerCase().trim().replace(/\s/g, '')
```

Comment:
Do we want to replace all whitespace? What if someone searches for "der Verwaltung" and then this can't be found because "derverwaltung" is not in the title?
Oh, maybe we need to think about doing different searches for every word. Otherwise "verwaltung für bla" might have a hit while "verwaltung bla" doesn't.